gobblin-core/src/main/java/org/apache/gobblin/source/extractor/hadoop/AvroFileSource.java

```diff
@@ -28,13 +28,16 @@
 
 import com.google.common.collect.Lists;
 
+import lombok.extern.slf4j.Slf4j;
+
 import org.apache.gobblin.configuration.ConfigurationKeys;
 import org.apache.gobblin.configuration.State;
 import org.apache.gobblin.configuration.WorkUnitState;
 import org.apache.gobblin.source.extractor.Extractor;
 import org.apache.gobblin.source.extractor.filebased.FileBasedSource;
 
 
+@Slf4j
 public class AvroFileSource extends FileBasedSource<Schema, GenericRecord> {
   private static final Logger LOGGER = LoggerFactory.getLogger(AvroFileSource.class);
```

Comment:
Can we clean up the LOGGER here if we are using the Slf4j wrapper?

Reply:
nice catch! Done.